tools/launchdev.sh

```diff
@@ -212,6 +212,8 @@ function st2start(){
     if [ "$copy_examples" = true ]; then
         echo "Copying examples from ./contrib/examples to $PACKS_BASE_DIR"
         cp -Rp ./contrib/examples $PACKS_BASE_DIR
+        cp -Rp ./st2tests/st2tests/fixtures/packs/dummy_pack_7 $PACKS_BASE_DIR
```

Comment:
We have a `test` pack for our integration tests @ https://github.com/StackStorm/st2tests/tree/master/packs/tests.  Can we use this instead of dummy_pack_7?

Reply:
Only issue by using this one is we have to put ` st2tests/st2tests/fixtures/packs/configs/dummy_pack_7.yaml` file under `st2tests/tests/configs/test.yaml`

Reply:
Why is this an issue adding a new config schema in the `tests` pack?

Reply:
Thought it is not normal way to place pack config file. 
Did change dummy_7_pack to `tests` and please review it. See https://github.com/StackStorm/st2/pull/4570/commits/c81f04270d3f7f442652c1adf84e38287968ef95
Thanks!